plugins/inventory/digital_ocean.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Per-Arne <per@sysx.no>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+import json
+
+from ansible.module_utils.urls import open_url
+
+DOCUMENTATION = r'''
+    name: digitalocean
```

Comment:
```suggestion
    name: digital_ocean
```

Reply:
I know "DigitalOcean" is the name of the company, and they seem to prefer it written that way. But if the plugin is named `digital_ocean`, the `name` should match. On the flip side, maybe this plugin should be called `digitalocean.py` instead of `digital_ocean.py`?

It makes no major difference to _me_, one way or the other. I just agree that it should be consistent at least inside any particular plugin.